helm/values.schema.json

```diff
@@ -0,0 +1,1856 @@
+            "type": "array",
+            "title": "The extraContainers schema",
+            "description": "Extra containers to be injected into the Connaisseur Pod to be used as sidecar.",
+            "examples": [
+              [
+                {
+                  "name": "exampleContainer",
+                  "image": "exampleImage:1.1.1",
+                  "volumeMounts": {
+                    "mountPath": "/exmaplepath/",
+                    "name": "example-vol"
+                  },
+                  "command": "examplecommand"
+                }
+              ]
+            ],
+            "additionalItems": true,
+            "items": {
+              "$id": "#/properties/deployment/properties/extraContainers/items",
+              "type": "object"
+            }
+          },
+          "extraVolumes": {
+            "$id": "#/properties/deployment/properties/extraVolumes",
+            "type": "array",
+            "title": "The extraVolumes schema",
+            "description": "Extra volumes to be added to the Connaisseur Pod.",
+            "examples": [
+              [
+                {
+                "name": "example-vol",
+                "emptyDir": {}
+                }
+              ]
+            ],
+            "additionalItems": true,
+            "items": {
+              "$id": "#/properties/deployment/properties/extraVolumes/items",
+              "type": "object"
+            }
+          },
+          "extraVolumeMounts": {
+            "$id": "#/properties/deployment/properties/extraVolumeMounts",
+            "type": "array",
+            "title": "The extraVolumeMounts schema",
+            "description": "Extra volume mounts of the Connaisseur Pod.",
+            "examples": [
+              [
+                {
+                  "mountPath": "/examplepath/",
+                  "name": "refresher-vol",
+                  "readOnly": "true"
+                }
+              ]
+            ],
+            "additionalItems": true,
+            "items": {
+              "$id": "#/properties/deployment/properties/extraVolumeMounts/items",
+              "type": "object"
+            }
+          }
+        },
+        "additionalProperties": true
+      },
+      "service": {
+        "$id": "#/properties/service",
+        "type": "object",
+        "title": "The service schema",
+        "description": "Configuration of the Connaisseur Kubernetes service resource.",
+        "examples": [
+          {
+            "type": "ClusterIP",
+            "port": 443
+          }
+        ],
+        "required": [
+          "type",
+          "port"
+        ],
+        "properties": {
+          "type": {
+            "$id": "#/properties/service/properties/type",
+            "type": "string",
+            "title": "The type schema",
+            "enum": [
+              "ClusterIP",
+              "NodePort",
+              "LoadBalancer",
+              "ExternalName"
```

Comment:
didn't we think that only `ClusterIP` is really a valid setting here? Could be restricted at this point, right?

Reply:
As this could be potentially breaking, we didn't include this here but took it up in https://github.com/sse-secure-systems/connaisseur/issues/496 :-)